loadtest/scripts/validator_failures.py

```diff
@@ -0,0 +1,120 @@
+        subprocess.check_output([seid_staking_cmd], stderr=subprocess.STDOUT,
+                                shell=True)
+    time.sleep(random.randint(600, 3600))
+    # Unbond
+    for validator in delegations:
+        print(f"Unbonding {10 * total_delegation} from {validator}")
+        seid_unbond_cmd = f"printf '12345678\n' seid tx staking unbond {validator} {delegations[validator]}usei --from admin -b block -y --chain-id {chain_id}"
+        subprocess.check_output([seid_unbond_cmd], stderr=subprocess.STDOUT,
+                                shell=True)
+
+
+def slow_network(validators, ssh_key, num_concurrent):
+    """
+    Slows the network of multiple validators at a time by adding 200ms latency.
+    Removes the latency after 10 mins to 1 hour.
+    """
+    i = 0
+    # Loop through all validators
+    while i < len(validators):
+        j = 0
+        slowed_validators = []
+        # Restart num_concurrent validators
+        while j < num_concurrent:
+            print(f"Slowing validators {validators[i + j]} by adding tc rules")
+            slowed_validators.append(validators[i + j])
+            tc_cmd = f"ssh -i {ssh_key} ubuntu@{validators[i + j]} 'sudo -S -p \"\" tc qdisc add dev ens5 root netem delay 200ms 10ms 25%'"
+            subprocess.check_output(tc_cmd, stderr=subprocess.STDOUT,
+                                    shell=True)
+            j += 1
+        time.sleep(random.randint(600, 3600))
+        print(f"Removing tc rules from validators {slowed_validators}")
+        for validator in slowed_validators:
+            tc_cmd = f"ssh -i {ssh_key} ubuntu@{validator} 'sudo -S -p \"\" tc qdisc del dev ens5 root'"
+            subprocess.check_output(tc_cmd, stderr=subprocess.STDOUT,
+                                    shell=True)
+        i += num_concurrent
+
+
+def _get_admin_acc():
+    seid_query_cmd = "printf '12345678\n' seid keys list --output json"
+    accs_output = json.loads(
+        subprocess.check_output([seid_query_cmd], stderr=subprocess.STDOUT,
+                                shell=True))
+    return filter(lambda x: x['name'] == 'admin', accs_output)['address']
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Runs various failure scenarios on validators. Note that delegation change must be run on genesis validator")
+    parser.add_argument('-s', '--ssh-key', type=str,
+                        help='Ssh key for validator nodes')
+    parser.add_argument('-f', '--failure', type=str,
+                        help='Type of failure to run. Should be one of ["restart", "delegation-change", "slow-network"]')
+    parser.add_argument('-v', '--validators', type=str,
+                        help='Comma separated list of validator nodes')
+    parser.add_argument('-c', '--chain-id', type=str, help='Chain id')
+    args = parser.parse_args()
+    assert args.failure in ["restart", "delegation-change",
```

Comment:
I think we can use `choices`

https://stackoverflow.com/questions/15836713/allowing-specific-values-for-an-argparse-argument